### ISSUE TITLE
During import, the configuration is logged at DEBUG level, including password values.

### ISSUE THREAD

**Issue by csassaman-els** [checkmarx-ts/checkmarx-python-sdk#92]
This behavior is new since upgrading from `0.5.5` to `0.6.3`:

After executing:

```python
from CheckmarxPythonSDK.CxRestAPISDK import AccessControlAPI
```

Several log lines are generated as the configuration is loaded. This includes the actual password value being used to authenticate.

```
2022-12-07 08:08:43,900 - CheckmarxPythonSDK - DEBUG - default config value: {'base_url': 'http://localhost:80', 'username': 'Admin', 'password': 'Password', 'grant_type': 'password', 'scope': 'sast_rest_api', 'client_id': 'resource_owner_client', 'client_secret': '014DF517-39D1-4453-B7B3-9930C563627C', 'scan_preset': 'Checkmarx Default', 'configuration': 'Default Configuration', 'team_full_name': '/CxServer', 'max_try': 3, 'report_folder': None, 'timeout': 60, 'verify': False, 'cert': None}
2022-12-07 08:08:43,900 - CheckmarxPythonSDK - DEBUG - config from config.ini file: {}
```

Is there a way to avoid logging sensitive data when importing the module? The `Bearer` token is also captured in the logs

Thanks!

**Comment by cx-happy-yang** [comment]
Hi @csassaman-els ,

Thanks for reporting the issue. Will fix it today.

**Comment by cx-happy-yang** [comment]
Hi @csassaman-els ,

It is fixed in version 0.6.4. By default it will not log debug message.

If you need to log debug message, you can use command line option --cx_debug true .

Best regards.
